blik/depictors/napari/particledepictor.py

```diff
@@ -40,27 +40,34 @@ def depict(self, rescale=True):
                                      edge_width=self.vector_widths[ax],
                                      )
 
-    def get_positions(self):
-        return self.datablock.positions.zyx * self.rescale * self.datablock.pixel_size
+    def _pad_to_ndim(self, array, value):
```

Comment:
should probably add a docstring here explaining exactly what the padding is doing, what should `value` be in specific situations etc

not super easy to parse the method just by reading it

Reply:
Yes, I'm not sure about this method. It might just be better to split it.

Reply:
I just opted to comment it properly.

Reply:
sorry, I still don't really understand what exactly 'value' refers to from your docstring/the method

Reply:
OH, I thought value was to do with the amount of padding! maybe call it 'fill_value' or something?

Reply:
my other question would be 'to match the total amount of dimensions' of what? 🙂

Reply:
I must have forgotten half a sentence :P Added some more comments...